OpenCover.UI.TestDiscoverer/NUnit/NUnitDiscoverer.cs

```diff
@@ -111,7 +135,11 @@ private TestMethod[] DiscoverTestsInClass(TypeDefinition type, TestClass @class)
 			foreach (var method in type.Methods)
 			{
 				bool isTestMethod = false;
-				var trait = new List<string>();
+                var traits = new List<string>();
+                if (@class.Traits != null && @class.Traits.Length > 0)
+                {
+                    traits.AddRange(@class.Traits);
+                }
```

Comment:
I apologize in advance if this is a dumb question, but how do you actually run the tests?  In my Visual Studio 2015 (Pro) Test Explorer, all I see is one test "RegularTestMethod" (which appears to be an MSTest test).  I'm looking for the NUnit tests like Discover_Finds_Regular_Test_Fixture_And_Method, and whatever new one I add to test the trait discovery.